File: C++/3174. Clear Digits.cpp
class Solution {
public:
    string clearDigits(string s) {
        vector<char>a;
        for(char c:s){
            if(c<='9'&&c>='0')
            a.pop_back();
            else
            a.push_back(c);
        }
        string k="";
        for(char c:a)
        k+=c;
        return k;
    }
};

File: JAVA/3174. Clear Digits.java
class Solution {
    public String clearDigits(String s) {
        List<Character>a=new ArrayList<>();
        for(int i=0;i<s.length();i++){
            if(s.charAt(i)>='0'&&s.charAt(i)<='9')
            a.remove(a.size()-1);
            else
            a.add(s.charAt(i));
        }
        String k="";
        for(int i=0;i<a.size();i++){
            k+=a.get(i);
        }
        return k;
    }
}